test/query/shorthand.test.ts

```diff
@@ -297,34 +427,19 @@ describe('query/shorthand', () => {
       assert.equal(str, 'a,q,sort={"field":"a","op":"mean","order":"descending"}');
     });
 
-    it('should return correct fieldDefShorthand string for bin with sort field definition object', () => {
```

Comment:
You're changing code for parsing -- I don't understand why you're removing code for shorthand generation.  
If the reason is the old example was wrong, the solution is definite NOT to remove them.